File: src/main/java/ar/edu/unq/spring/service/interfaces/PersonajeService.java
package ar.edu.unq.spring.service.interfaces;

import ar.edu.unq.spring.modelo.Personaje;

import java.util.Optional;

public interface PersonajeService {
    Iterable<Personaje> allPersonajes();
    void guardarPersonaje(Personaje personaje);
    Optional<Personaje> recuperarPersonaje(Long personajeId);
    void clearAll();
}

File: src/main/kotlin/ar/edu/unq/spring/service/interfaces/PersonajeService.kt
package ar.edu.unq.spring.service.interfaces

import ar.edu.unq.spring.modelo.Personaje

/* Eviten el Spanglish :D */
interface PersonajeService {
    fun allPersonajes(): Collection<Personaje>
    fun guardarPersonaje(personaje: Personaje)
    fun recuperarPersonaje(personajeId: Long): Personaje?
}